### ISSUE TITLE
Handle warnings as errors disabled

### ISSUE THREAD

**Issue by mberz** [pyfar/gallery#2]
Due to TOC issues in the pyfar_demo.ipynb notebook I've disabled treating sphinx warnings as errors on CircleCI.

Needs to be re-enabled once the demo notebook has been replaced by changing line 76 in `.circleci/config.yml`

```
make html SPHINXOPTS="-W"
```

**Comment by f-brinkmann** [comment]
Yes, the TOC is causing it. However, the notebook will be removed in favor of more specific notebooks. Marco opened the issue to make sure we don't forget removing the exception later.
